hugegraph-server/hugegraph-hstore/src/main/java/org/apache/hugegraph/backend/store/hstore/HstoreStore.java

```diff
@@ -390,8 +397,8 @@ public Iterator<BackendEntry> query(Query query) {
     //                for (ConditionQuery conditionQuery :
     //                    ConditionQueryFlatten.flatten(cq)) {
     //                    Id label = conditionQuery.condition(HugeKeys.LABEL);
-    //                 /* 父类型 + sortKeys： g.V("V.id").outE("parentLabel").has
-    //                 ("sortKey","value")转成 所有子类型 + sortKeys*/
+    //                 /* 父类型 + sortKeys：g.V("V.id").outE("parentLabel").has
+    //                 ("sortKey","value") 转成 所有子类型 + sortKeys*/
```

Comment:
In this PR, the code style here should not be adjusted for now. It will be adjusted together later.